packages/composites/crypto-volatility-index/src/config/index.ts

```diff
@@ -2,7 +2,7 @@ import { Requester } from '@chainlink/ea-bootstrap'
 import { Config } from '@chainlink/ea-bootstrap'
 
 export const DEFAULT_NETWORK = 'ETHEREUM'
-export const NAME = 'CRYPTO_VOLATILITY_INDEX'
+export const NAME = 'CVI'
```

Comment:
Is this safe?

Reply:
It makes it match what we have in RDD, to help us with EA Telemetry alerting. I am not aware of where this would cause an issue personally.